diff --git a/node_modules/@portabletext/svelte/dist/defaultComponents/DefaultBlock.svelte b/node_modules/@portabletext/svelte/dist/defaultComponents/DefaultBlock.svelte
index 0167807..7567edd 100644
--- a/node_modules/@portabletext/svelte/dist/defaultComponents/DefaultBlock.svelte
+++ b/node_modules/@portabletext/svelte/dist/defaultComponents/DefaultBlock.svelte
@@ -6,7 +6,7 @@ $: style = value.style || 'normal';
 {#if ['h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'blockquote'].includes(style)}
   <svelte:element this={style}><slot /></svelte:element>
 {:else if style === 'normal'}
-  <p><slot /></p>
+  <p class="py-3"><slot /></p>
 {:else}
-  <slot />
+  <p class="py-3"><slot /></p>
 {/if}
